client/app/telecom/pack/xdsl/modem/connectedDevices/pack-xdsl-modem-connectedDevices.controller.js

```diff
@@ -71,7 +72,6 @@ angular.module('managerApp').controller('XdslModemConnectedDevicesCtrl', functio
   self.refresh = function () {
     this.devices = null;
     this.loading = true;
-    TucPackXdslModemMediator.disableCapabilities();
```

Comment:
Wasn't it useful though ?

Reply:
No it generates some bad returns on the page on refresh modem connected devices.

Reply:
Can the function be deleted then ? :)

Reply:
Yes it is was useless here. It works fine, and it's used elsewhere in the complete page for modem components.

Reply:
Awesome thank you